includes/admin/settings.php

```diff
@@ -133,17 +363,65 @@ function options_page() {
 	<div class="wrap">
 		<h1><?php esc_html_e( 'Autoshare for Twitter', 'autoshare-for-twitter' ); ?></h1>
 
-		<form action='options.php' method='post'>
-			<?php
-			settings_fields( AT_GROUP );
-			do_settings_sections( 'autoshare-for-twitter' );
-			submit_button();
-			?>
-		</form>
+		<div class="autoshare-settings">
+			<form action='options.php' method='post'>
+				<?php
+				settings_fields( AT_GROUP );
+				do_settings_sections( 'autoshare-for-twitter' );
+				submit_button();
+				?>
+			</form>
+			<div class="brand">
+				<a href="https://10up.com" class="logo" title="<?php echo esc_html__( '10up', 'autoshare-for-twitter' ); ?>">
+					<img src="<?php echo esc_url( trailingslashit( AUTOSHARE_FOR_TWITTER_URL ) . 'assets/images/10up.svg' ); ?>" alt="<?php echo esc_html__( '10up logo', 'autoshare-for-twitter' ); ?>" />
+				</a>
+				<p>
+					<strong>
+						<?php echo esc_html__( 'Autoshare for Twitter', 'autoshare-for-twitter' ) . ' ' . esc_html__( 'by', 'autoshare-for-twitter' ); ?> <a href="https://10up.com" class="logo" title="<?php echo esc_html__( '10up', 'autoshare-for-twitter' ); ?>"><?php echo esc_html__( '10up', 'autoshare-for-twitter' ); ?></a>
+					</strong>
+				</p>
+				<nav>
+					<a href="https://github.com/10up/autoshare-for-twitter#installation" target="_blank" title="<?php echo esc_attr__( 'Installation & Set up', 'autoshare-for-twitter' ); ?>">
```

Comment:
Probably can remove this link as the plugin is already installed and we're adding set up instructions via this PR